maestral/sync.py

```diff
@@ -2696,58 +2705,50 @@ def get_remote_folder(self, dbx_path: str = "/") -> List[SyncEvent]:
 
             dbx_path = dbx_path or "/"
             is_dbx_root = dbx_path == "/"
-            results = []
+            success = True
 
             if is_dbx_root:
                 logger.info("Downloading your Dropbox")
             else:
                 logger.info("Downloading %s", dbx_path)
 
-            if any(is_child(folder, dbx_path) for folder in self.excluded_items):
-                # if there are excluded subfolders, index and download only included
-                recursive = False
```

Comment:
Did removing this disable excluded folders?

Reply:
This only includes excluded folders in indexing, they will still be skipped during the download. The reasoning behind this was a performance hit when recursing many sub-folders instead of indexing all at once. However, this needs to be balanced with the gains when a large number of folders have been excluded.

I'll see if there is a better way of handling this.